assoc_plots.R

```diff
@@ -0,0 +1,68 @@
+#! /usr/bin/env Rscript
+library(argparser)
+
+argp <- arg_parser("Association plots")
+argp <- add_argument(argp, "assoc_file",
+                     help = "Association test results file (.rds)")
+argp <- add_argument(argp, "--out_prefix", help = "Prefix for output files",
+                     default = "")
+argv <- parse_args(argp)
+
+sessionInfo()
+print(argv)
+library(dplyr)
+library(ggplot2)
+library(RColorBrewer)
+# Get pipeline directory
+initial.options <- commandArgs(trailingOnly = FALSE)
+file.arg.name <- "--file="
+script.name <- sub(file.arg.name, "", initial.options[grep(file.arg.name, initial.options)])
+script.basename <- dirname(script.name)
```

Comment:
I'm not crazy about this, but it is working when executed from the command line. The reason something like this needs to be here is because there's a function definition in utils.R and this script needs to know where it's saved on disk. If there are better ways to do this, lmk

Reply:
Option 1 is to just paste the function definition into this script, if we're not planning to use it anywhere else. Option 2 is to create an R package for this pipeline, like TopmedPipeline. I might start with (1), and move to (2) if we start having more cases like this.